slasher/db/kv/spanner_new.go

```diff
@@ -0,0 +1,110 @@
+package kv
+
+import (
+	"context"
+	"errors"
+
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/slasher/detection/attestations/types"
+	bolt "go.etcd.io/bbolt"
+	"go.opencensus.io/trace"
+)
+
+var errWrongSize = errors.New("wrong data length for min max span byte array")
+var highestObservedValidatorIdx uint64
+
+// GetValidatorSpan unmarshal a span from an encoded, flattened array.
+func (es *EpochStore) GetValidatorSpan(ctx context.Context, idx uint64) (types.Span, error) {
+	ctx, span := trace.StartSpan(ctx, "slasherDB.getValidatorSpan")
```

Comment:
I'm not sure if we should trace here, this seems like it would be a commonly called function and it has a rather constant execution time, the span might take up more flame than the function.

Reply:
removed